lib/emoji.js

```diff
@@ -66,32 +98,54 @@ Emoji._get = function _get(emoji) {
  * @return {string}
  */
 Emoji.get = function get(emoji) {
-  emoji = trim(emoji);
+  emoji = stripColons(emoji);
 
   return Emoji._get(emoji);
 };
 
 /**
+ * find the emoji by either code or name
+ * @param {string} nameOrCode The emoji to find, either `coffee`, `:coffee:` or `☕`;
+ * @return {object}
+ */
+Emoji.find = function get(nameOrCode) {
```

Comment:
can you rename the function to `find` (could be confusing to see `get` in the stack trace when this is the `find` function)?

we should always ensure that the internal function name (`function functionName`) is the same as the API function name (`Emoji.functionName`):

```js
Emoji.functionName = function functionName (arguments) {
```